lib/llvm-ir.ts

```diff
@@ -174,6 +176,9 @@ export class LlvmIrParser {
             filters.push(this.functionAttrs);
             lineFilters.push(this.attributeAnnotation);
         }
+        if (options.filterComments) {
+            filters.push(this.commentRe);
```

Comment:
I don't know if a comment can ever be emitted at the end of a line instead of on its own line, but at the moment this will filter any line containing a comment anywhere which seems risky. Sorry about the naming here too, filters vs lineFilters could probably be made more clear.

Reply:
I don't think it actually comes up in anything LLVM can output, but yes I was intending to handle comments at the end of the line since they're syntactically valid. I'll update the patch to use `lineFilters` instead.

Reply:
Ah, I see. I need a separate regex for each